src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
```

Comment:
We don't pass health value to constructor, all new objects should have health of 100
```suggestion
  constructor(name) {
```

Reply:
you didn't fix Illia's comment